### PR TITLE
SImplefied travis config and added composer scripts

### PR DESCRIPTION
**Pull request by Nyholm**
Simplify the travis config and make sure you can run all relevant tests by `composer test`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,5 +31,8 @@
             "email": "travis@mailgunhq.com"
         }
     ],
-    "minimum-stability": "stable"
```
Comment (DavidGarciaCat) [inline composer.json]: Why to remove this line?

**Comment by Nyholm** [comment]
You are correct. Thanks

File: composer.json (hunk 1 above)
Comment (DavidGarciaCat) [inline composer.json]: You're right, forget my comment

**Comment by pirogoeth** [comment]
I dig. Merged.

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**